3rdparty/python/requirements.txt

```diff
@@ -10,7 +10,7 @@ fasteners==0.16.3
 freezegun==1.2.1
 ijson==3.1.4
 packaging==21.3
-pex==2.1.163
+pex==2.2.1
```

Comment:
I note that the PR description says 2.2.2, not exactly matching this.

Reply:
Embarrassing typo fixed.